vue/components/organisms/pickers/pickers.vue

```diff
@@ -16,6 +16,18 @@
                 v-on:click="slideLeftParts"
             />
             <ul class="parts-container" ref="partsPicker">
+                <li
+                    class="part button button-part button-extra"

```

Comment:
Not actually a `part`, right? Is this needed because of style?

Reply:
Yes, its there because of the style